portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,24 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
```

Comment:
[put imported names in ASCII order](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing)

Reply:
Updated with commit b849b6c

Reply:
Updated again with commit a0c479e